### ISSUE TITLE
uploader token error

### ISSUE THREAD

**Issue by heni40** [enzienaudio/heavy#174]
Hello together,
with the old uploader Script everything worked fine when using the commandline promts under windows. But after switching machines and having to use the new hv_uploader script installed via pip I cant figure out a way to make it work.
No matter what I do I get this error:

> The user token is invalid. Generate a new one at https://enzienaudio.com/h/'username'/settings/.
> [91mError:[0m (<type 'exceptions.IndexError'>) list index out of range
> Getting a weird error? Get the latest version with 'pip install hv-uploader -U', or check for issues at https://github.com/enzienaudio/heavy/issues.

I tried the --clear_token command and entering a new generated token from the settings menu, tried that a couple of times with no luck. 

Any suggestions?
Thanks

**Comment by mhroth** [comment]
@heni40 in cases like this, are you _sure_ that you copied the full token correctly? Otherwise, try manually deleting the file at ~/.heavy/token.

**Comment by heni40** [comment]
yes, I'm sure I copied it in full and correctly. Deleting the toke file unfortunately just makes the script ask for a token again. After pasting the token I get the same error message.

**Comment by mhroth** [comment]
Could you send us the token file to info@enzienaudio.com ? I'll take a look.

**Comment by heni40** [comment]
hm, it just worked... must have been the shortcut for pasting to the cmd promt. 
Now I feel bad for making an issue out of that. I's always the same, fiddeling around for hours and researching on git hub... and after you post something it works.

One last thing. The --out command does not seem to work as expected. 
Although using it just like in the Doc:

> hv-uploader "C:\path\to\patch directory" --out "C:\path\to\output" --name heavy

It only creates the files on the website. What am I doing wrong now?

**Comment by mhroth** [comment]
No worries at all :) We're here to help. You're not doing anything wrong, but you do need to tell the uploader what to download. For instance, if you want to download the C source files, pass the `-g c-src` flag.

You can get a complete list of files that you can download with `hv-uploader --help`.
